### PR TITLE
KVM2 driver docs.

### PR DESCRIPTION
**Pull request by dlorenc**
cc @r2d4

### CODE REVIEW

File: docs/drivers.md [hunk 1]
```diff
@@ -9,11 +9,45 @@ extra binary to be present in the host PATH.
...
+# Fedora/CentOS/RHEL
+$ newgrp libvirt
+```
+
+Then install the driver itself:
+
+```
+curl -LO https://storage.googleapis.com/minikube/releases/latest/docker-machine-driver-kvm2 && chmod +x docker-machine-driver-kvm2 && sudo mv docker-machine-driver-kvm2 /usr/bin/
+```
+
```
Comment (r2d4) [inline docs/drivers.md]: It might be worth it to add a line here on how to use it

`--vm-driver=kvm2`

**Comment by zakame** [comment]
Hi!

Do you plan to keep the KVM2 driver in minikube's tree, or will this be spun off its own repo later on?  I'm comparing against @dhiltgen's original driver, and there are many PRs already there that extend it (e.g. using KVM storage pools, using virtio, etc.) that perhaps could be ported to KVM2 soon.

**Comment by zakame** [comment]
@dhiltgen thanks, I'm using your driver with some of the PRs applied fairly frequently at my work, so I was a bit concerned about the state here.

@dlorenc I took a closer look, and seem like the kvm2 driver is still very minikube-specific (e.g. it pulls the minikube iso instead of boot2docker.)

I'd be happy to help out here in any case or form the KVM support takes for docker-machine/minikube.

**Comment by abitrolly** [comment]
Are there any hints why KVM2 is better?

**Pull request merged**